stubs/python-xlib/Xlib/protocol/rq.pyi

```diff
@@ -0,0 +1,416 @@
+from _typeshed import ReadableBuffer, SliceableBuffer, SupportsTrunc
+from array import array
+from collections.abc import Callable, Iterable, Sequence
+
+# mypy thinks type is List.type https://github.com/python/mypy/issues/14205
+from typing import Any, SupportsInt, Type, TypeVar, overload, type_check_only  # noqa: Y022
```

Comment:
You can avoid the `noqa` by doing `from builtins import type as Type`

(Also as @JelleZijlstra commented on the mypy issue, I think mypy's behaviour is in fact correct here :)

Reply:
Yep, saw that comment. Didn't feel intuitive to me, but since this is expected behavior I'll update the comment and I'll go with your suggested workaround.